jobs/pacman-qualys-enricher/src/main/java/com/tmobile/cso/pacman/qualys/jobs/Util.java

```diff
@@ -344,10 +349,15 @@ public static Map<String, String> fetchEniMacInfo() {
         LOGGER.info("fetchEniMacInfo payLoad: {}",payLoad);
         List<Map<String, String>> data = new ArrayList<>();
         String scrollId = fetchDataAndScrollId(endPoint, data, payLoad);
-        do {
-            endPoint = SCROLL_URI + scrollId;
-            scrollId = fetchDataAndScrollId(endPoint, data, null);
-        } while (scrollId != null);
+
+        while (scrollId != null) {
+            endPoint = "/_search/scroll";
+            payLoad = "{\n" +
+                    "  \"scroll\" : \"2m\", \n" +
+                    "  \"scroll_id\" : \"%s\" \n" +
+                    "}";
+            scrollId = fetchDataAndScrollId(endPoint, data, String.format(payLoad, scrollId));
+        }
```

Comment:
This looks like duplicate code, please extract to a separate function

Reply:
done.